zerver/lib/actions.py

```diff
@@ -817,6 +873,7 @@ def check_send_message(*args, **kwargs):
     return do_send_messages([message])[0]
```

Comment:
OK, cool; this one may be worth opening a `mypy` issue about since I think it's an interesting issue that they may run into as well.  I could imagine e.g. there being some fancy syntax to handle this like `type: copy[check_message]`